test/lib/code_corps_web/controllers/password_reset_controller_test.exs

```diff
@@ -29,7 +36,21 @@ defmodule CodeCorpsWeb.PasswordResetControllerTest do
     {:ok, _} = AuthToken.changeset(%AuthToken{}, current_user) |> Repo.insert
     attrs = %{"token" => "random token", "password" => "123456", "password_confirmation" => "123456"}
     conn = post conn, password_reset_path(conn, :reset_password), attrs
+
     assert json_response(conn, 404)
   end
 
+  test "does not create resource and renders errors when error in token timeout occurs", %{conn: conn} do
+    timeout = Application.get_env(:code_corps, :password_reset_timeout)
+    Application.put_env(:code_corps, :password_reset_timeout, 0)
+
+    current_user = insert(:user)
+    {:ok, auth_token} = AuthToken.changeset(%AuthToken{}, current_user) |> Repo.insert
+    attrs = %{"token" => auth_token.value, "password" => "123456", "password_confirmation" => "123456"}
+    conn = post conn, password_reset_path(conn, :reset_password), attrs
+    assert json_response(conn, 404)
+
+    Application.put_env(:code_corps, :password_reset_timeout, timeout)
```

Comment:
There's a new test helper for changing the env you should be able to use now.

Reply:
@JoshSmith Fixed